cmd/restore.go

```diff
@@ -38,6 +42,8 @@ type restoreCommand struct {
 	verbose       bool
 	loggingConfig string
 	backupFile    string
+
+	connectFunc func() (core.Database, func(), error)
```

Comment:
Database includes a Close method, I don't think we need a closer func returned here. If it's just to get rid of the error from Database.Close, you could change Database.Close not to have an error - the underlying mgo.Session.Close doesn't return anything anyway.